### PR TITLE
Brighten default/affiliate dark star sub badge in dark mode

### LINKED ISSUE #2541: Affiliate sub badge invisible in dark mode
Like the title says, the Affiliate sub badge is invisible in dark mode, because it's almost the same shade of black/grey

### PR DESCRIPTION
**Pull request by Stoupedog**
Fixes #2541

### CODE REVIEW

**Comment by night** [comment]
how does default twitch render it on their dark chat?

**Comment by Stoupedog** [comment]
They don't even change it at all, they use exactly the same image as light mode so you can barely see it.

image

File: src/css/betterttv-dark.css [hunk 1]
```diff
@@ -58,13 +58,18 @@ body {
     background: #111 !important;
 }
 
+/* brighten default dark star sub badge */
+.ember-chat .badges img.badge[src="https://static-cdn.jtvnw.net/badges/v1/19dd8673-124d-4f44-830c-b0f4f9d78635/1"] {
```
Comment (night) [inline src/css/betterttv-dark.css]: match on `.badges a[href*="/subscribe"] img.badge` instead

Comment (Stoupedog) [inline src/css/betterttv-dark.css]: That brightens every subscriber badge including custom ones uploaded by partnered streamers

Comment (night) [inline src/css/betterttv-dark.css]: u r right, my bad

**Pull request merged**